server/website/website/utils.py

```diff
@@ -494,3 +495,23 @@ def model_to_dict2(m, exclude=None):
         if fname not in exclude:
             d[fname] = getattr(m, fname, None)
     return d
+
+
+def check_and_run_celery():
+    celery_status = os.popen('python manage.py celery inspect ping').read()
+    if 'OK' in celery_status:
+        return 'celery is running'
+
+    retries = 0
+    while retries < 5:
+        LOG.warning('Celery is not running.')
+        retries += 1
+        call_command('stopcelery')
+        os.popen('python manage.py startcelery &')
+        time.sleep(5 * retries)
```

Comment:
I try to restart celery on my local machine, it takes about 30 seconds to finish restarting. 
I think we can set the interval to 30s ?   
<code> time.sleep(30)  </code>

Reply:
Thanks!